bpf/lib/nodeport.h

```diff
@@ -28,6 +28,9 @@
 #include "fib.h"
 #include "srv6.h"
 
+DECLARE_CONFIG(__u32, nodeport_dev_ifindex, "ifindex of the device to use for nodeport egress")
+#define NATIVE_DEV_IFINDEX CONFIG(nodeport_dev_ifindex) /* Backwards compatibility */
+
```

Comment:
Did you mean to replace `DIRECT_ROUTING_DEV_IFINDEX` ? The `NATIVE_DEV_IFINDEX` is really just the ifindex for the device that the BPF program is attached to, I don't see why it would need to be configurable.

Reply:
:bulb: so it's just the naming / description that's a bit too focused on the nodeport usage.

Reply:
As discussed offline, this should actually be THIS_INTERFACE_IFINDEX. I'll fold `NATIVE_DEV_IFINDEX` into `THIS_INTERFACE_IFINDEX` and make it a runtime config defined in node_config.h.